packages/s/sunshine/package.yml

```diff
@@ -1,15 +1,17 @@
 name       : sunshine
-version    : 0.23.1
-release    : 15
+version    : 2025.122.141614
+release    : 16
 source     :
-    - git|https://github.com/LizardByte/Sunshine.git : v0.23.1
+    - git|https://github.com/LizardByte/Sunshine.git : v2025.122.141614
 license    : GPL-3.0-only
 component  : network.web
 homepage   : https://github.com/LizardByte/Sunshine
 summary    : Gamestream host for Moonlight
 description: |
     Open source implementation of NVIDIA's GameStream, as used by the NVIDIA Shield. It can be used with any Moonlight client.
     Sunshine gets configured through a web UI.
+# Set necessary env so the correct version is displayed in the logs and UI 
+environment: export BUILD_VERSION=v2025.122.141614 BRANCH=master COMMIT=""
```

Comment:
Could you see if changing this to `BUILD_VERSION=$version` works? That way who ever updates this package won't have to remember to change it every time.

Reply:
Nice, that works! I added it to the documentation: https://github.com/getsolus/help-center-docs/pull/611